notebooks/search/09-semantic-text.ipynb

```diff
@@ -0,0 +1,527 @@
+    "\n",
+    "Let's populate the index with our example dataset of 12 movies."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "2b5a46b60660a489"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "url = \"https://raw.githubusercontent.com/elastic/elasticsearch-labs/main/notebooks/search/movies.json\"\n",
+    "response = urlopen(url)\n",
+    "movies = json.loads(response.read())\n",
+    "\n",
+    "operations = []\n",
+    "for movie in movies:\n",
+    "    operations.append({\"index\": {\"_index\": \"semantic-text-movies\"}})\n",
+    "    operations.append(movie)\n",
+    "client.bulk(index=\"semantic-text-movies\", operations=operations, refresh=True)"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "24f0133923553d28"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Semantic Search\n",
+    "\n",
+    "Now that our index is populated, we can query it using semantic search.\n",
+    "\n",
+    "### Aside: Pretty printing Elasticsearch search results\n",
+    "\n",
+    "Your `search` API calls will return hard-to-read nested JSON.\n",
+    "We'll create a little function called `pretty_search_response` to return nice, human-readable outputs from our examples."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "6fff5932fcbac1b0"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "def pretty_search_response(response):\n",
+    "    if len(response[\"hits\"][\"hits\"]) == 0:\n",
+    "        print(\"Your search returned no results.\")\n",
+    "    else:\n",
+    "        for hit in response[\"hits\"][\"hits\"]:\n",
+    "            id = hit[\"_id\"]\n",
+    "            score = hit[\"_score\"]\n",
+    "            title = hit[\"_source\"][\"title\"]\n",
+    "            runtime = hit[\"_source\"][\"runtime\"]\n",
+    "            plot = hit[\"_source\"][\"plot\"]\n",
+    "            keyScene = hit[\"_source\"][\"keyScene\"]\n",
+    "            genre = hit[\"_source\"][\"genre\"]\n",
+    "            released = hit[\"_source\"][\"released\"]\n",
+    "            \n",
+    "            pretty_output = f\"\\nID: {id}\\nScore: {score}\\nTitle: {title}\\nRuntime: {runtime}\\nPlot: {plot}\\nKey Scene: {keyScene}\\nGenre: {genre}\\nReleased: {released}\"\n",
+    "            \n",
+    "            print(pretty_output)"
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "ad417b4b3f50c889"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Semantic Search with the `semantic` Query\n",
+    "\n",
+    "We can use the `semantic` query to quickly & easily query the `semantic_text` field in our index.\n",
+    "Under the hood, an embedding is automatically generated for our query text using the `semantic_text` field's inference endpoint.\n",
+    "\n",
+    "TODO: Add link to `semantic` query documentation"
```

Comment:
Will fix once we know how documentation links should be handled (see [this comment](https://github.com/elastic/elasticsearch-labs/pull/271/files#r1642950911))